Jenkinsfile

```diff
@@ -112,7 +112,7 @@ pipeline {
                     s3upDocker {
                         localArtifact = "build/dai-deployment-templates-${version}.zip"
                         artifactId = 'dai-deployment-templates'
-                        version = version
+                        version = ${version}
```

Comment:
it needs to be a string like `version = "${version}"`.

Reply:
Just added quote